src/components/screens/home-screen/home-screen.tsx

```diff
@@ -119,6 +125,20 @@ class HomeScreen extends React.Component<Props, State> {
     );
   }
 
+  /**
+   * Method for greeting message
+   */
+  private greeting = () =>{
```

Comment:
```suggestion
  private generateGreeting = () => {
```
Spaces and more descriptive name